deploy/cluster_role.yaml

```diff
@@ -41,4 +52,22 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
\ No newline at end of file
+  - '*'
+- apiGroups:
+  - rbac.authorization.k8s.io
+  - authorization.openshift.io
+  resources:
+  - rolebindings
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - delete
+- apiGroups:
+  - rbac.authorization.k8s.io
```

Comment:
As we already figured out that, if someone is trying to bind clusterrole from group ` authorization.openshift.io`, then it'll fail, I think we should add clusterrole of group `authorization.openshift.io` to avoid any future issues with Rolebinding which is trying bind ClusteRole belongs to group ` authorization.openshift.io`